`jetpack watch` produces a bunch of error output

### Impacted plugin

Jetpack

### Steps to Reproduce

1. Run `jetpack watch`.
2. See a whole bunch of error output in its full glory.

Discussion in p1655132549395029-slack-CDLH4C1UZ

```
[extensions] ERROR in ./extensions/shared/components/upgrade-nudge/style.scss (./extensions/shared/components/upgrade-nudge/style.scss.webpack[javascript/auto]!=!../../../node_modules/.pnpm/css-loader@6.5.1_webpack@5.72.1/node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[2].use[1]!../../../node_modules/.pnpm/postcss-loader@6.2.0_yokl3cbwhadljnh75p6ymh6bce/node_modules/postcss-loader/dist/cjs.js??ruleSet[1].rules[2].use[2]!../../../node_modules/.pnpm/sass-loader@12.4.0_sass@1.38.1+webpack@5.72.1/node_modules/sass-loader/dist/cjs.js!./extensions/shared/components/upgrade-nudge/style.scss)
[extensions] Module Error (from ../../../node_modules/.pnpm/postcss-loader@6.2.0_yokl3cbwhadljnh75p6ymh6bce/node_modules/postcss-loader/dist/cjs.js):
[extensions] Loading PostCSS "postcss-custom-properties" plugin failed: Cannot find module 'postcss-custom-properties'
[extensions] Require stack:
```

<img width="1911" alt="image" src="https://user-images.githubusercontent.com/36432/173387979-eb5d4835-f9a8-4710-83dd-6d76d0d09233.png">

### A clear and concise description of what you expected to happen.

_No response_

### What actually happened

_No response_

### Other information

_No response_

### Operating System

_No response_

### OS Version

_No response_

### Browser

Other / Not applicable

### Browser Version(s)

_No response_

--- a/projects/plugins/jetpack/changelog/fix-jetpack-watch
+++ b/projects/plugins/jetpack/changelog/fix-jetpack-watch
@@ -1,0 +1,5 @@
+Significance: patch
+Type: other
+Comment: Fix `watch` command. No change to the plugin.
+
+

--- a/projects/plugins/jetpack/package.json
+++ b/projects/plugins/jetpack/package.json
@@ -39,7 +39,7 @@
 		"test-client": "NODE_PATH=tests:_inc/client jest --config=tests/jest.config.client.js",
 		"test-extensions": "TZ=UTC jest --config=tests/jest.config.extensions.js",
 		"test-gui": "NODE_PATH=tests:_inc/client jest --config=tests/jest.config.gui.js",
-		"watch": "concurrently --names client-js,client-css,masterbar,extensions,widget-visibility 'webpack watch --config ./tools/webpack.config.js' 'webpack watch --config ./tools/webpack.config.css.js' 'webpack watch --config ./tools/webpack.config.masterbar.js' 'webpack watch --config ./tools/webpack.config.extensions.js' 'webpack watch --config ./tools/webpack.config.widget-visibility.js'"
+		"watch": "NODE_PATH=\"$PWD/node_modules\" concurrently --names client-js,client-css,masterbar,extensions,widget-visibility 'webpack watch --config ./tools/webpack.config.js' 'webpack watch --config ./tools/webpack.config.css.js' 'webpack watch --config ./tools/webpack.config.masterbar.js' 'webpack watch --config ./tools/webpack.config.extensions.js' 'webpack watch --config ./tools/webpack.config.widget-visibility.js'"
 	},
 	"browserslist": [
 		"extends @wordpress/browserslist-config"

--- a/projects/plugins/social/changelog/fix-jetpack-watch
+++ b/projects/plugins/social/changelog/fix-jetpack-watch
@@ -1,0 +1,5 @@
+Significance: patch
+Type: fixed
+Comment: Fix `watch` command. No change to the plugin.
+
+

--- a/projects/plugins/social/package.json
+++ b/projects/plugins/social/package.json
@@ -20,7 +20,7 @@
 		"build-production-concurrently": "pnpm run clean && concurrently 'NODE_ENV=production BABEL_ENV=production pnpm run build-client' && pnpm run validate",
 		"clean": "rm -rf build/",
 		"validate": "pnpm exec validate-es build/",
-		"watch": "pnpm run build && webpack watch"
+		"watch": "pnpm run build && NODE_PATH=\"$PWD/node_modules\" webpack watch"
 	},
 	"browserslist": [
 		"extends @wordpress/browserslist-config"

